cmd/podman/containers/create.go

```diff
@@ -117,6 +117,12 @@ func commonFlags(cmd *cobra.Command) error {
 	if err != nil {
 		return err
 	}
+	if cmd.Flags().Changed("read-only-tmpfs") {
+		cliVals.ReadWriteTmpFS, err = flags.GetBool("read-only-tmpfs")
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
This change seems unnecessary, you can just keep the flag definition as BoolVar().